pulp_rpm/app/kickstart/treeinfo.py

```diff
@@ -55,6 +55,37 @@ def load(self, f):
         except MissingSectionHeaderError:
             raise TypeError(_("Treeinfo file should have INI format"))
 
+    def sections_from_general(self, parser):
+        """
+        Write sections based on general.
+        """
+        general = parser._sections.get("general")
+        sections = parser._sections.keys()
+
+        if general and "release" not in sections:
+            release = {}
+            translate = {"family": "name", "version": "version"}
+            for key, new_key in translate.items():
+                if general.get(key):
+                    release[new_key] = general[key]
+            if release:
+                parser._sections["release"] = release
+
+        if general and "tree" not in sections:
+            tree = {}
+            fake_timestamp = "".join([str(ord(a)) for a in general["name"].replace(" ", "")])
```

Comment:
this is super dirty!
the treeinfo file didn't have a `timestamp`
http://download.opensuse.org/distribution/leap/15.1/repo/oss/.treeinfo
```
[header]
version = 1.0

[release]
name = openSUSE Leap
version = 15.1

[general]
arch = x86_64
family = openSUSE Leap
name = openSUSE Leap 15.1
version = 15.1
platforms = x86_64,xen

[images-x86_64]
kernel = boot/x86_64/loader/linux
initrd = boot/x86_64/loader/initrd

[images-xen]
kernel = boot/x86_64/loader/linux
initrd = boot/x86_64/loader/initrd
```
depending on the header version, productmd validates if timestamp is integer or float.
I needed something unique by treeinfo file, and it should be a number, so I took the general.name (in this case is: `openSUSE Leap 15.1`), and replaced the letter by numbers

Reply:
@fabricio-aguiar , naive question. Can we just set timestamp to 0 if it's not available? 
Why do we need something unique?

Reply:
I think it is not naive, actually it is much better than what I did. I thought on uniqueness because of unique_together, but putting a 0 would solve it